src/rgw/rgw_http_client.cc

```diff
@@ -69,36 +69,49 @@ struct rgw_http_req_data : public RefCountedObject {
 /*
  * the simple set of callbacks will be called on RGWHTTPClient::process()
  */
```

Comment:
"hermitization" means encapsulation?  to do with the matrices?  to do with thierry l'hermite?